mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -92,16 +92,24 @@ storage:
           Volume=/dev/virtio-ports/testisocompletion:/mnt/serial
           PodmanArgs=--privileged
           Network=host
+          LogDriver=passthrough
           Exec=shell -- kola qemuexec --netboot /mnt/workdir-tmp/boot.ipxe --usernet-addr 10.0.3.0/24 -- -device virtio-serial -chardev file,id=iscsi-completion-virtio,path=/mnt/serial,append=on -device virtserialport,chardev=iscsi-completion-virtio,name=testisocompletion
           [Install]
           # Start by default on boot
           WantedBy=multi-user.target
           [Service]
+          # We want ExecStartPost to be run after the podman container have actually started
+          # The default type (simple) runs post commands immediately after the main service process has been forked off
+          # rather than waiting for the exit code
+          Type=oneshot
```

Comment:
since we now don't need `ExecStartPost` do we need `Type=oneshot` still?

If so we should at least not mention `ExecStartPost` in the comment here.

Reply:
good catch. Thanks